api/dto/target.go

```diff
@@ -140,8 +154,20 @@ func TargetVerification(targets []string, ttl time.Duration, isRemote bool) []Tr
 	return functionsOfTargets
 }
 
+// HaveTreesError checks that at least one node of tree has a problem with error type.
+// It is wrapper to handle slice of trees.
+func HaveTreesError(trees []TreeOfProblems) bool {
```

Comment:
Mabe this is better?

```suggestion
func DoTreesHaveErrors(trees []TreeOfProblems) bool {
```

Reply:
`DoTreesHaveError` is fine, not error**s**, it is singular. Maybe it is need to rename to `isSomeTreeHasError` to stress that it checks that at least one error in all trees

Reply:
I think `do` stresses the fact that it is a predicate well enough. Let's name it `DoesAnyTreeHaveError`. `Any` is someting commonly used in predicates, so it will be easy to read

Reply:
`DoesAnyTreeHaveError` is nice, @kissken do you agree?

Reply:
pff, ok

Reply:
Done https://github.com/moira-alert/moira/pull/817/commits/7c68d2f125f4275b5d23fc8573896cbc67fc3f9c